meta-mender-tegra/meta-mender-tegra-jetpack5/recipes-bsp/tegra-binaries/tegra-redundant-boot/nv_update_verifier.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=OTA bootloader update verifier
+After=nvstartup.service
```

Comment:
https://github.com/OE4T/meta-tegra/blame/b146e186ffb927f8c3f2ee3efa7e29cd49d313ba/recipes-bsp/tegra-binaries/tegra-redundant-boot/nv_update_verifier.service#L3

Missing `dev-disk-by\x2dpartlabel-kernel.device` here compared to the upstream.  Not sure if that was intentional.